website/docs/developer/deployment-guide/deployment-with-cumulus.md

```diff
@@ -413,7 +429,7 @@ the ingest workflow.
   }
 },
   "Type":"Task",
-  "Resource":"module.orca.orca_lambda_copy_to_archive_arn",
+  "Resource":"module.orca.orca_lambda_copy_to_archive_adapter.arn",
```

Comment:
Should be a variable that points to the cumulus adapter. May need to look at their TF stuff to provide the path

Reply:
Will reference to `https://github.com/nasa/cumulus/blob/master/tf-modules/cumulus/outputs.tf#L86` for users to add the value.